tests/conftest.py

```diff
@@ -47,9 +41,6 @@ def local_expected_csvs(test_data_path):
         grades = f.readlines()
     with open(test_data_path / "expected/enrollments.csv", 'r') as f:
         enrolments = f.readlines()
-    with open(test_data_path / "expected/oneroster_demographics.csv",
-              'r') as f:
-        or_demographics = f.readlines()
```

Comment:
Let's go ahead and remove the text expect file to at `tests/data/expected/oneroster_demographics.csv`.